### PR TITLE
Add a .github repo to sandbox

### PR DESCRIPTION
**Pull request by mattmoor**
This is to start defining common metadata across the organization.

Related: https://github.com/knative/community/issues/218

/assign @grantr @evankanderson @tcnghia @markusthoemmes 

cc @chaodaiG @chizhg @coryrc @n3wscott 

/hold

I'd like a couple pairs of eyeballs on this to sanity check me 😇

### CODE REVIEW

File: peribolos/knative-sandbox.yaml [hunk 1]
```diff
@@ -220,6 +220,14 @@ orgs:
     repos:
       # NOTE: If you are adding a repo here, you may also want to grant repo
       # permissions for one or more teams, in the next section.
+      .github:
+        allow_merge_commit: false
+        allow_rebase_merge: false
+        # See also: https://github.blog/changelog/2019-02-21-organization-wide-community-health-files/
+        # See also: https://docs.github.com/en/actions/configuring-and-managing-workflows/sharing-workflow-templates-within-your-organization
+        description: "Repository for sharing org-wide Github metadata and workflows."
```
Comment (n3wscott) [inline peribolos/knative-sandbox.yaml:228]: workflow templates. It seems you don't auto-apply all the workflows from this repo unless I misunderstand it

Comment (n3wscott) [reply peribolos/knative-sandbox.yaml:228]: maybe mention issue templates too?

Comment (mattmoor) [reply peribolos/knative-sandbox.yaml]: done

Comment (mattmoor) [reply peribolos/knative-sandbox.yaml]: that's metadata :)

**Comment by n3wscott** [comment]
lgtm but I want @evankanderson to take a look

**Comment by chaodaiG** [comment]
lgtm to me

**Comment by mattmoor** [comment]
Ok, if @evankanderson or @tcnghia want to weight in, I'll merge and we can try something trivial to test the waters.

**Comment by chizhg** [comment]
lgtm for me as well.

File: peribolos/knative-sandbox.yaml [hunk 2]
```diff
@@ -351,7 +359,10 @@ orgs:
           general project administration.
         privacy: closed
         repos:
+          # All repos should be listed here!
```
Comment (evankanderson) [inline peribolos/knative-sandbox.yaml:362]: As an FYI, being org admins should make this redundant.

Comment (mattmoor) [reply peribolos/knative-sandbox.yaml:362]: I do not think that the Prow bots nor the release leads are org admins

**Pull request merged**